src/app/pages/Project.tsx

```diff
@@ -68,14 +117,50 @@ function Docs() {
         <h2>new pack</h2>
         <DropUpload
           fileLimit={5e7}
-          fileActionButtonFunc={handleAdd}
+          fileActionButtonFunc={(file: FileWithPath[]) =>
+            handleAdd(file, 'single_pack')
+          }
           fileActionButtonText={'ADD'}
           mimeType="application/json"
           // Do not support zip now.
           // mimeType='application/json, application/x-rar-compressed, application/octet-stream, application/zip, application/octet-stream, application/x-zip-compressed, multipart/x-zip'
           allowMultiple={true}
         />
       </div>
+
+      {projectInfo && projectInfo.project_type === 'crossdoc' ? (
+        <div className="content_left" style={{marginLeft: '20px'}}>
+          <h2>All multi docs:</h2>
+          {crossdocs
+            ? crossdocs.map(d => (
+                <ul key={d.id}>
+                  <li>
+                    <Link to={`/crossdocs/${d.id}`}>{d.name}</Link>{' '}
+                    <button onClick={() => handleDelete(d.id, 'multi_pack')}>
+                      X
+                    </button>
+                  </li>
+                </ul>
+              ))
+            : 'Empty'}
+        </div>
+      ) : null}
+      {projectInfo && projectInfo.project_type === 'crossdoc' ? (
```

Comment:
this check is the same as the one above, can we combine them?

Reply:
Combined into a single one in the newer commit.